packages/sdks/src/blocks/video/component-info.ts

```diff
@@ -57,6 +57,12 @@ export const componentInfo: ComponentInfo = {
       defaultValue: 'cover',
       enum: ['contain', 'cover', 'fill', 'auto'],
     },
+    {
+      name: 'preload',
+      type: 'text',
+      defaultValue: 'auto',
```

Comment:
```suggestion
      defaultValue: 'metadata',
```

Wdyt if we match the spec, default value in browsers is `metadata` https://developer.mozilla.org/en-US/docs/Learn/Performance/video#video_preload

Reply:
okay cool